### PR TITLE
fix: update TopsortError model, rename to Error

### PR DESCRIPTION
**Pull request by roalonso**
This PR is part of a series of changes related to our error codes:
- https://github.com/Topsort/auction-engine/pull/1019
- https://docs.topsort.com/reference/errors (updated) ✅

### CODE REVIEW

File: topsort-api-v2.yml [hunk 1]
```diff
@@ -355,10 +359,13 @@ components:
             - missing_slots
             - no_products
             - no_purchase_items
+            - purchase_item_quantity_less_or_equal_than_zero
+            - resolved_bid_id_not_found
```
Comment (PabloReszczynski) [inline topsort-api-v2.yml:363]: Do we have a custom error code for timeout related errors?

Comment (roalonso) [reply topsort-api-v2.yml:363]: Actually now I'm not sure. We might have a mix of both, but anyways, there's no timeout-specific error, I assume we return an internal_server_error in that case? @sk-

**Review by sk- — CHANGES_REQUESTED** [review]

File: topsort-api-v1.yml [hunk 2]
```diff
@@ -674,10 +675,12 @@ components:
             - missing_slots
             - no_products
             - no_purchase_items
+            - purchase_item_quantity_less_or_equal_than_zero
+            - resolved_bid_id_not_found
             - too_few_impressions
             - too_few_slots
```
[the remark this answers is not in the document]
Comment (sk-) [reply topsort-api-v1.yml:681]: My bad, didn't realized you were modifying both files.

File: topsort-api-v2.yml [hunk 3]
```diff
@@ -355,12 +366,20 @@ components:
...
             - too_few_impressions
             - too_few_slots
+            - too_many_auctions
         docUrl:
-          enum: ['https://topsort-api.redoc.ly']
+          type: string
```
Comment (sk-) [inline topsort-api-v2.yml:375]: Consider adding `format: uri`

**Pull request merged**